src/models/dish.ts

```diff
@@ -72,9 +72,9 @@ const DishSchema = new Schema<IDish>({
     },
     ratingsAverage: {
         type: Number, 
-        min: [1, 'Rating must be equal or greater than 1'], 
+        min: [0, 'Rating must be equal or greater than 1'], 
```

Comment:
can you plz update the msg as well, sorry for that again 
`  min: [0, 'Rating must be equal or greater than 0'], `